doc.go

```diff
@@ -37,8 +37,21 @@ What we needed/wanted was the following(and more):
 
 Install
 
-Go install:
-  go get -u github.com/c2fo/vfs/v5
+Pre 1.17:
+  go get -u github.com/c2fo/vfs/v6
+
+Post 1.17:
+  go install -u github.com/c2fo/vfs/v6
+
+
+Upgrading
+
+Upgrading from v5 to v6
+With v6.0.0, we changed the Auth creation options to accept an array of Key Exchange algorithms rather than a string.
```

Comment:
Can you change (any anywhere else this may appear)
```
With v6.0.0, we changed the Auth creation options to
```
to
```
With v6.0.0, sftp.Options struct changed to
```
I think it makes it sound more statement-of-fact rather than a narrative and now is explicit that this only affects sftp.

Reply:
Added in this commit https://github.com/C2FO/vfs/pull/102/commits/cceabb4a1c595dd10f67efa33a3c49b30c75baaa